components/AppBar/src/MDCAppBarContainerViewController.h

```diff
@@ -59,4 +59,30 @@
 /** The content view controller to be displayed behind the header. */
 @property(nonatomic, strong, nonnull, readonly) UIViewController *contentViewController;
 
+#pragma mark - Enabling top layout guide adjustment behavior
+
+/**
+ If enabled, the content view controller's top layout guide will be adjusted as the flexible
+ header's height changes and the content view controller view's frame will be set to the container
+ view controller's bounds.
+
+ This behavior is disabled by default, but it will be enabled by default in the future. Consider
+ enabling this behavior and making use of the topLayoutGuide in your view controller accordingly.
```

Comment:
`topLayoutGuide` is deprecated in iOS 11. Should we be moving teams to `safeAreaLayoutGuide.topAnchor` instead?

Reply:
We still need to support topLayoutGuide for pre-iOS 11 devices.

Reply:
Agreed, but I'm thinking that we should not actively guide them to a deprecated solution. Can we push them to safeAreaLayoutGuide.topAnchor and maybe have a header-only comment (not docs) telling them to use topLayoutGuide as a fall-back?

Reply:
I can do this in a follow-up PR.

Reply:
https://github.com/material-components/material-components-ios/issues/4374